### ISSUE TITLE
Two files to update because of @angular/cli

### ISSUE THREAD

**Issue by KevinEverywhere** [keathmilligan/angular2-d3-v4#3]
One line change in package.json, four lines in angular-cli.json.
[Thanks, the rest works well.]
------------------------------------------------------------------------
package.json:
------------------------------------------------------------------------
Unable to find "@angular/cli" in devDependencies.

The package "angular-cli" has been deprecated and renamed to "@angular/cli".

Please take the following steps to avoid issues:
"npm uninstall --save-dev angular-cli"
"npm install --save-dev @angular/cli@latest" 

------------------------------------------------------------------------
angular-cli.json:
------------------------------------------------------------------------
Environment configuration does not contain "environmentSource" entry.

A new environmentSource entry replaces the previous source entry inside environments.

To migrate angular-cli.json follow the example below:

Before:

"environments": {
  "source": "environments/environment.ts",
  "dev": "environments/environment.ts",
  "prod": "environments/environment.prod.ts"
}

After:

"environmentSource": "environments/environment.ts",
"environments": {
  "dev": "environments/environment.ts",
  "prod": "environments/environment.prod.ts"
}

**Comment by katestearns-suplari** [comment]
I had the same issue

**Comment by iposton** [comment]
I had the same problem. I am using angular version 4.0.0 and my local machine uses node version 6.10.2. I manually changed <code>"angular-cli": "1.0.0-beta.24"</code> to <code>"@angular/cli": "1.0.0"</code> in package.json nested in devDependencies. 

Also I had to change the enviromentSource in angular-cli.json posted above by @KevinEverywhere. Use this code. 

```js
"environmentSource": "environments/environment.ts",
"environments": {
  "dev": "environments/environment.ts",
  "prod": "environments/environment.prod.ts"
}

```
